test/packages/good/data_stream/foo/fields/external-fields.yml

```diff
@@ -3,4 +3,7 @@
   description: Event family
   fields:
     - name: category
-      external: ecs
\ No newline at end of file
+      external: ecs
+    - name: module
```

Comment:
nit: could you please put `event.module` in a different `fields.yml`? The intention of `external-fields.yml` was to keep only referenced ones :) something like `ecs.yml` would be good.

Reply:
Absolutely, I should have realized by the name that this wasn't the right place. I moved it to `some_fields.yml` and made the example generic.